modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -50,7 +50,7 @@ Before you create a secure properties file and define its properties, review the
 
 == Create a Secure Configuration Properties File
 
-The first task in securing configuration properties is to create a `.yaml` configuration file or a Spring-formatted `.properties` file, where you will define the properties. MuleSoft recommends using a YAML configuration file, because it allows the addition of type validations and autocompletion. The Mule Secure Configuration Properties Extension module enables you to configure these `.yaml` or `.properties` file types. +
+The first task in securing configuration properties is to create a `.yaml` configuration file or a Spring-formatted `.properties` file, where you define the properties. The Mule Secure Configuration Properties Extension module enables you to configure these `.yaml` or `.properties` file types. +
```

Comment:
Kim clarified for me just last week in an edit that we should spell out file types instead of using <.type>.  YAML instead of .yml, etc. What I am doing is something like this: "...is to create a YAML configuration file '.yml` or a..."
Also, instead of .yaml, it should be yml.

Reply:
Change "these" to "the" in the last sentence.